lib/Dialect/FIRRTL/Transforms/ModuleSummary.cpp

```diff
@@ -0,0 +1,99 @@
+//===- ModuleSummaryt.cpp ---------------------------------------*- C++ -*-===//
```

Comment:
```suggestion
//===- ModuleSummary.cpp ----------------------------------------*- C++ -*-===//
```
(not 100% on dash count, just added one because removed a char)